Canvas/CanvasUITests/Login/LoginTests.swift

```diff
@@ -17,140 +17,83 @@
 import XCTest
 import TestsFoundation
 
-enum LoginStart {
-    static var findMySchool: Element {
-        return app.find(label: "Find my school")
-    }
-
-    static func previousUser(name: String) -> Element {
-        return app.find(label: name)
-    }
-}
-
-enum LoginFindSchool: String, ElementWrapper {
-    case searchField
-
-    static func resultItem(for name: String) -> Element {
-        return app.find(label: name)
-    }
-}
-
-enum CanvasLogin {
-    static var emailTextField: Element {
-        return XCUIElementWrapper(app.webViews.textFields["Email"])
-    }
-
-    static var passwordTextField: Element {
-        return XCUIElementWrapper(app.webViews.secureTextFields["Password"])
-    }
-
-    static var logInButton: Element {
-        return XCUIElementWrapper(app.webViews.buttons["Log In"])
-    }
-}
-
-enum RyanaLogin {
-    static var ldapButton: Element {
-        return XCUIElementWrapper(app.webViews.staticTexts["LDAP"])
-    }
-}
-
 class LoginTests: CanvasUITests {
+    override var user: UITestUser? { return nil }
 
     func testFindSchool() {
-        // Find my school
-        LoginStart.findMySchool.tap()
+        LoginStart.findSchoolButton.tap()
         LoginFindSchool.searchField.typeText("mtech")
-        LoginFindSchool.resultItem(for: "MTECH").waitToExist()
+        LoginFindAccountResult.item(host: "mtec.instructure.com").waitToExist()
     }
 
     func testCanvasLoginToDashboard() {
-       loginUser(username: "student1", password: "password")
+        logInUser(.readStudent1)
```

Comment:
nitpick: Is there an extra space in front here?

Reply:
there wasn't enough space before. It was only indented 3 from `func`.